.github/workflows/playwright.yml

```diff
@@ -0,0 +1,165 @@
+name: Playwright Tests
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  pull_request:
+  push:
+    branches:
+      - master
+
+env:
+  TERM: xterm
+  GO_VERSION: 1.19.6
+  NODE_VERSION: 16.15.0
+  MM_RUDDER_WRITE_KEY: ${{ secrets.MM_RUDDER_PLUGINS_PROD }}
```

Comment:
@trilopin I'm wondering what the benefit is for providing a telemetry key in testing. Doesn't this skew our telemetry numbers? Maybe we should have a test key to use? I could be missing some context here

Reply:
indeed, this shouldn't be our prod write key

I'd rather keep that empty and let dev key to automatically be assigned